receiver/splunkhecreceiver/receiver.go

```diff
@@ -58,11 +58,11 @@ var (
 	errInvalidMethod          = errors.New("invalid http method")
 	errInvalidEncoding        = errors.New("invalid encoding")
 
-	okRespBody                = initJSONResponse(responseOK)
```

Comment:
Why not use `initJSONResponse` any more?

Reply:
It doubles encodes what is now a valid JSON object into a JSON string.

Reply:
Sigh. I should decouple this into another PR.

Reply:
done.